src/pods/footer/footer.component.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import classes from './footer.pod.module.css';
+import { useCanvasViewSettingsContext } from '@/core/providers';
+
+export const FooterComponent: React.FC = () => {
+  const { filename } = useCanvasViewSettingsContext();
+
+  const isNewDocument = () => (filename ? filename : 'New document');
```

Comment:
Let's avoid magic strings on the middle of the code, create a const on line 4 and add something like:

```ts
const NEW_DOCUMENT_NAME = "New Document";
```

When you use _is...._ you assume it's a boolean (true/false), let's avoid this in order to get the code more readable:

```diff
- const isNewDocument = () => (filename ? filename : 'New document');
- const documentName = () => (filename ? filename : NEW_DOCUMENT_NAME);
```